### ISSUE TITLE
Thắc mắc

### ISSUE THREAD

**Issue by ZtopVN** [dominhhai/dominhhai.github.io#11]
Em chào anh
Em đã đọc một số bài về deep learning của anh. Trong bài về cài đặt RNN có đoạn nhân ma trận em không hiểu. Tại sao lại là U.xt+ W.st-1. Mà U(100x8000), x(1x8000), W và s cũng vậy. Nguyên tắc là các ma trận này đâu thể nhân với nhau. Mong anh giải đáp. Em xin cám ơn

**Comment by dominhhai** [comment]
x khi implement với Python là dạng vec-tơ hàng nhé

**Comment by ZtopVN** [comment]
vâng. cám ơn anh. em đang xây dựng mạng neuron cho việc kiểm tra ngữ pháp
tiếng anh thì mô hình rnn hoặc lstm có phù hợp không anh
nếu được thì a có thể giúp em không

2018-03-24 8:11 GMT+07:00 Do Minh Hai <notifications@github.com>:
